### DISCUSSION TITLE
無人曳引機🌾

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#191] [General]
**新聞內容：**
John Deere新的無人曳引機Tractor支援TruSet鑿犁、GPS導引系統等先進系統，有6對立體攝影鏡頭，可360度偵測障礙物，並且計算距離，攝影機擷取的圖像，會交由深度神經網路進行運算，該模型會在100毫秒內，對每個畫素進行分類，依據偵側到障礙物與否，來決定曳引機的行為。

無人曳引機的使用很簡單，農民只需要將機器運送到田地，並且使用行動應用程式Operations Center Mobile，啟動自動操作即可，在無人曳引機運作時，農民可以離開田地執行其他任務，並透過行動裝置監控機器的狀態。

Operations Center Mobile供使用者即時存取視訊、圖像、資料和指標，讓農民能夠調整耕作速度和深度，出現任何運作上的問題，或是機器出現狀況，用戶也會遠端收到通知，並且即時介入調整。

**新聞心得：**
無人曳引機在美國或是其他大面積種植範圍的國家進行採收對於農民都是相當方便的，只要讓機器在田地上就會開始自動操作，就好像把自動化工業帶到農業上來應用一樣！只是這樣的機器感覺價格很高！

**新聞來源：**
https://www.ithome.com.tw/news/148725
![r4f167334_rrd_@1x](https://user-images.githubusercontent.com/90883526/148884697-4fc47691-6dd7-46a8-ad5d-5048c3e6879c.jpg)

**Comment by Timmy110403540** [comment]
感覺是推廣給美國那種大草地使用。在台灣可能使用率會比較小，因為土地小很多。

**Comment by wh20502** [comment]
一台80萬美，如果哪個農夫有買，我覺得不如省下來，把菜價再便宜一點...，拿去當PT薪水也綽綽有餘，可能要等COST DOWN台灣才有機會看到。

**Comment by ghost** [comment]
這個不˙太適合台灣這種農地小的 應該想別的東西來給台灣使用
